libcxx/src/experimental/stacktrace/osx/osx.h

```diff
@@ -0,0 +1,31 @@
+//===----------------------------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+#ifndef _LIBCPP_STACKTRACE_OSX_H
```

Comment:
```suggestion
#ifndef _LIBCPP_STACKTRACE_MACOS_H
```

Nit: I'd call this macOS.h since macOS isn't OSX for a while and this feature won't be available in any versions called OSX. So I think using "macOS" is more appropriate.